apps/minifront/vite.config.ts

```diff
@@ -5,6 +5,7 @@ import { commitInfoPlugin } from './src/utils/commit-info-vite-plugin';
 import polyfillNode from 'vite-plugin-node-stdlib-browser';
 
 export default defineConfig({
+  define: { 'globalThis.__DEV__': 'import.meta.env.DEV' },
```

Comment:
Instead of defining a custom DEV property, just using the globally available `import.meta.env.MODE` feels more standard

Reply:
`import.meta.env.MODE` and `import.meta.env.DEV` are part of vite-env, and aren't generally available in places where we might like to have a dev mode switch, such as every package

it's typical to just refer to process.env.NODE_ENV but technically wrong, and since the packages target browser runtime instead of node runtime and don't import node types, it makes typescript upset

declaring a `__DEV__` global and then checking that, and requiring consumers to define it to indicate dev mode, is safer (won't error) and becoming a convention. it's used by apollo and react-native

https://www.apollographql.com/docs/react/development-testing/reducing-bundle-size/#why-use-globalthis__dev__-instead-of-processenvnode_env